### PR TITLE
Follow-up to PR39260: Add Image Registry Operator configuration parameter `disabledRegistry` 

### PR DESCRIPTION
**Pull request by bmcelvee**
Follow-up to https://github.com/openshift/openshift-docs/pull/39260. Adds content to the main branch to be cherry picked to 4.7+. 

SME and QE review tracked on https://github.com/openshift/openshift-docs/pull/39260. 

Preview link: https://deploy-preview-40417--osdocs.netlify.app/openshift-enterprise/latest/registry/configuring-registry-operator.html#registry-operator-configuration-resource-overview_configuring-registry-operator

### CODE REVIEW

File: modules/registry-operator-configuration-resource-overview.adoc [hunk 1]
```diff
@@ -61,6 +61,9 @@ for the route.
 |`replicas`
 |Replica count for the registry.
 
+|`disableRedirect`
+| `disableRedirect` controls whether to route all data through the registry, rather than redirecting to the backend. Defaults to false.
```
Comment (bergerhoffer) [inline modules/registry-operator-configuration-resource-overview.adoc:65]: * Should it just start with "Controls" to be more consistent with how the rest of the descriptions start?
* Should "backend" be "back end" (per ISG, as a noun)?
* Should "false" be in backticks?

**Review by bergerhoffer — COMMENTED** [review]
Just a few minor questions!

File: modules/registry-operator-configuration-resource-overview.adoc (hunk 1 above)
Comment (bmcelvee) [reply modules/registry-operator-configuration-resource-overview.adoc]: Thanks! Updated.

**Pull request merged**